src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/search.html.twig

```diff
@@ -1,23 +1,25 @@
-<div id="sidebar-search" class="hidden">
+<div id="sidebar-search" class="{{ render_hidden_by_default is not defined or render_hidden_by_default ? 'hidden' }}">
```

Comment:
using `render_hidden_by_default ?? true` might be easier to read than `render_hidden_by_default is not defined or render_hidden_by_default`